java/ql/lib/semmle/code/java/frameworks/apache/IOGenerated.qll

```diff
@@ -1,684 +1,828 @@
+        "org.apache.commons.io.output;FileWriterWithEncoding;true;FileWriterWithEncoding;(String,Charset);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;FileWriterWithEncoding;true;FileWriterWithEncoding;(String,Charset,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;FileWriterWithEncoding;true;FileWriterWithEncoding;(String,CharsetEncoder);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;FileWriterWithEncoding;true;FileWriterWithEncoding;(String,CharsetEncoder,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;FileWriterWithEncoding;true;FileWriterWithEncoding;(String,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;FileWriterWithEncoding;true;FileWriterWithEncoding;(String,String,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(File);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(File,Charset);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(File,Charset,boolean,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(File,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(File,String,boolean,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(File,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(File,boolean,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(String,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;LockableFileWriter;true;LockableFileWriter;(String,boolean,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;XmlStreamWriter;true;XmlStreamWriter;(File);;Argument[0];generated:create-file",
+        "org.apache.commons.io.output;XmlStreamWriter;true;XmlStreamWriter;(File,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io.serialization;ClassNameMatcher;true;matches;(String);;Argument[-1];generated:regex-use[0]",
+        "org.apache.commons.io.serialization;ClassNameMatcher;true;matches;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;CopyUtils;true;copy;(InputStream,Writer);;Argument[0];generated:write-file",
+        "org.apache.commons.io;CopyUtils;true;copy;(InputStream,Writer,String);;Argument[0];generated:write-file",
```

Comment:
InputStreams aren't writeable

Reply:
These were derived due to java.io.Writer handwritten write-file sink. I have removed this sink.